consutil/main.py

```diff
@@ -28,7 +29,7 @@ def show():
     devices = getAllDevices()
     busyDevices = getBusyDevices()
 
-    header = ["Line", "Baud", "PID", "Start Time"]
+    header = ["Line", "Actual/Configured Baud", "PID", "Start Time"]
```

Comment:
Maybe split Actual Baud and Configured Baud into separate columns?

Reply:
We have current show line output combine them in a same column.

Reply:
emm. the current show line shows tx/rx in the same column. I guess having 2 columns for configured would be ok then.

Reply:
I'm OK with whatever is decided to be most legible. I just thought it was worth a discussion.

Reply:
maybe shorten the title to baud/conf? it seems that the title is taking lots of horizontal space.

Reply:
I think "baud/conf" isn't very descriptive and can be confusing.

Reply:
Here's how two columns looks:
```
  Line    Actual Baud    Configured Baud    PID    Start Time
------  -------------  -----------------  -----  ------------
     0           9600               9600
     1           9600               9600
     2           9600               9600  23285  Wed Aug  8 00:49:02 2018
     3           9600                  -
```

I could also keep the current headers but align the dash:
```
  Line    Actual/Configured Baud    PID    Start Time
------  ------------------------  -----  ------------
     0      9600/9600
     1      9600/9600
     2      9600/9600             23285  Wed Aug  8 00:49:02 2018
     3      9600/-
```